### ISSUE TITLE
chromvar from signac wrapper is stuck (not progressing)

### ISSUE THREAD

**Issue by Dena2021** [GreenleafLab/chromVAR#98]
Hello, 
I would really appreciate your help. I am running chromvar from a Seurat object on 29K cells and 192K peaks.
R version 4.1.1 (2021-08-10)
chromVAR version 1.16.0
Signac version 1.6.0
Seurat version 4.1.0

It starts running fine:
> register(MulticoreParam(10, progressbar = TRUE))
> mySeurat <- RunChromVAR(
+   object = mySeurat,
+   genome = BSgenome.Mmusculus.UCSC.mm10,
+   verbose=TRUE
+ )
Computing GC bias per region
Selecting background regions
Computing deviations from background
  |                                                         |   0%

Basically, it gets stuck at this stage forever. 
Any suggestions?
Thanks in advance

**Comment by hsriniva1104** [comment]
Hi, I'm facing the same issue. I let my job run for almost 4 days and still nothing changed. I'd love some help here.

**Comment by Dena2021** [comment]
Before chromvar command run
register(SerialParam())
